src/lib/hooks/useAvatar.js

```diff
@@ -51,18 +51,18 @@ const useAvatar = avatar => {
 }
 
 export const useUploadedAvatar = () => {
```

Comment:
@johnsmith-gooddollar can you explain the logic of this method? why you use context, localstorage and ref?
add inline comments

Reply:
1. context is needed to get avatar from profile
2. avatarPassed / setAvatarPassed is avatar was uploaded
3. local storage is used to keep avatar just uploaded during navigation between screens (we have 2 screens on web - first for upload second for crop)

Reply:
I still dont understand why we need both
context + localstorage + ref
it is like 3 different places we store it at, why? can this be simplified?

Reply:
yes, by refactoring this component to load and crop image in a single screen
i could open a separate task for this because this is out of this PR's scope

Reply:
@sirpy https://app.zenhub.com/workspaces/gooddollar-5d50833888a83c6880d3e345/issues/gooddollar/gooddapp/3684 opened issue